habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -225,7 +225,7 @@ class WaypointControlActionConfig(VelocityControlActionConfig):
     # Action space range
     waypoint_lin_range: List[float] = [-0.5, 0.5]  # meters
     waypoint_ang_range: List[float] = [-np.pi, np.pi]  # radians
-    wait_duration_range: List[float] = [0.0, 10.0]  # seconds
+    wait_duration_range: List[float] = [0.1, 10.0]  # seconds
```

Comment:
We should change this to [0, 10.0] as the timestep is not guaranteed to be 0.1
I made changes in the branch `add_velocity` so we always run for at least 1 step.